Change Datatype in some fields

nhoyosp opened the pull request:
Changed some datatypes, added a new AI PK field in every table and changed the naming conception

nhoyosp committed:
Change Datatype in some fields

Changed some datatypes, added a new AI PK field in every table and changed the naming conception

jhonatan247 commented on ms-campus-kid-subjects/DataBase/Career.cs:
```diff
@@ -11,11 +11,13 @@ namespace DataBase
-        [ForeignKey("id_faculty")]
-        public virtual Faculty faculty { get; set; }
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id_Career { get; set; }
```
Could you use `camelCase` for attribute naming?
This is a standard that we are using in the project.

jhonatan247 commented on ms-campus-kid-subjects/DataBase/Migrations/20220929023932_InitDB.cs:
```diff
@@ -15,36 +16,39 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 columns: table => new
                 {
-                    id_site = table.Column<string>(type: "varchar(255)", nullable: false)
+                    Id_Site = table.Column<int>(type: "int", nullable: false)
```
is it possible to use `camelCase` for column naming?

nhoyosp committed:
Corrected the naming conventions

Corrected the naming conventions to camelCase. Also changed redundant model naming in most of the project

jhonatan247 commented on ms-campus-kid-subjects/DataBase/Career.cs:
```diff
@@ -11,10 +11,12 @@ namespace DataBase
-        public string id_faculty { get; set; }
-        [ForeignKey("id_faculty")]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int idCareer { get; set; }
```
```suggestion
        public int id { get; set; }
```

jhonatan247 replied:
Please apply this suggestion to all similar cases in your code.

jhonatan247 commented on ms-campus-kid-subjects/DataBase/Career.cs:
```diff
@@ -11,10 +11,12 @@ namespace DataBase
+        public int idCareer { get; set; }
+        [Column(TypeName = "varchar(64)")]
+        public string name { get; set; }
+        public int idFaculty { get; set; }
```
```suggestion
        public int facultyId{ get; set; }
```

jhonatan247 replied:
Please apply this suggestion to all similar cases in your code.

nhoyosp committed:
few naming corrections

Changed the order in the FK naming and  deleted the column posfix in the id columns.
Also modified the POST method.
